rasa_core/channels/slack.py

```diff
@@ -172,7 +197,8 @@ def process_message(self, on_new_message, text, sender_id):
         """
         retry_reason = request.headers.environ.get('HTTP_X_SLACK_RETRY_REASON')
         retry_count = request.headers.environ.get('HTTP_X_SLACK_RETRY_NUM')
-        if retry_count and retry_reason in self.errors_ignore_retry:
+        ## if retry_count and retry_reason in self.errors_ignore_retry:        
+        if retry_count:
```

Comment:
any reason why you changed the condition in this if statement?

Reply:
@EPedrotti nope that's a bug hunting adjustment that I did not remember to remove. 

Is there somewhere you are testing the code? I'm new to the project and have no idea where you are testing anything etc. As for formatting, I don't know what style checker you are using, but if you tell me I can make sure it passes anything you need. 

Best wishes

M

Reply:
@EPedrotti  delinted... are we done yet? :)